modules/bridge/src/main/ts/ephox/bridge/components/dialog/DialogFooterButton.ts

```diff
@@ -14,6 +14,7 @@ interface BaseDialogFooterButtonSpec {
   primary?: boolean;
```

Comment:
This needs the `@deprecated` comment as well.
```suggestion
  /** @deprecated use `buttonType: "primary"` instead */
  primary?: boolean;
```

Reply:
I thought to write a test for this but I saw [this test](https://github.com/tinymce/tinymce/blob/496a2b9662f9988e5cde4900c9ad67961cee7bc0/modules/tinymce/src/themes/silver/test/ts/headless/components/dialogbutton/DialogButtonTest.ts#L11) which seems to cover `buttonType` property. Should there be another one specific for dialog footer button?

Reply:
If that test covered this then it would have been failing... if you look closer at it then it only handles dialog buttons, not dialog footer buttons.

Reply:
Figured out the difference. I am writing tests as we speak..